controls/radspreadsheet/features/command-descriptors.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Command Descriptors
+page_title: Command Descriptors
+description: Describes how you can override the default command descriptors and create custom commands 
```

Comment:
Describes how you can override the default command descriptors of RadSpreadsheet for WPF and create custom commands.

Reply:
Fixed